bin/compile

```diff
@@ -80,7 +80,7 @@ info "node_modules cached: $modules_cached"
 
 echo ""
 
-info "NPM_CONFIG_PRODUCTION=$NPM_CONFIG_PRODUCTION"
```

Comment:
There are several tests to ensure env vars are working:

https://github.com/heroku/heroku-buildpack-nodejs/blob/master/bin/test#L202-L255

This just prints the environment variables as debug lines; it doesn't change anything.